test/ssltestlib.c

```diff
@@ -12,6 +12,13 @@
 #include "internal/nelem.h"
 #include "ssltestlib.h"
 #include "testutil.h"
+#ifndef OPENSSL_NO_KTLS
+#include "internal/ktls.h"
```

Comment:
Style nit: Inside a "#if" so insert a space between "#" and "include"

Reply:
Fixed.